src/components/Login.tsx

```diff
@@ -0,0 +1,70 @@
+import { useState } from 'react';
+import Button from './Button';
+import ErrorCard from './ErrorCard';
+import Input from './Input';
+
+export default function Login() {
+  const [error, setError] = useState(null);
+  const [credentials, setCredentials] = useState({
+    email: '',
+    password: '',
+  });
```

Comment:
Can we adopt Formik for this? Im sure we want to validate email format and required fields. Also handle submission and form state.